pyproject.toml

```diff
@@ -52,7 +52,7 @@ style = "pep440"
 
 [tool.black]
 line-length = 100
-target-version = ["py38", "py39", "py310"]
+target-version = ["py38", "py39"]
```

Comment:
Why drop 3.10?

Reply:
Black does not recognize `py310` apparently.

Reply:
I think tox had an old version of black. It works now ... ran `tox --recreate` and the error disappeared. It was a local issue.